src/components/CreateOrUpdateRequest.svelte

```diff
@@ -84,9 +81,9 @@ function imageUploaded(event) {
 }
 
 async function cancelRequest() {
-  await cancel(params.id)
+  await cancel($params.id)
```

Comment:
Should this be `await cancel($params.requestId)`?

Reply:
Yes, good catch!